src/app/shared/tile-header/tile-header.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { ActionName, DropdownModel } from 'src/app/shared/tile-header/dropdown.model';
+
+@Component({
+  selector: 'app-tile-header',
+  templateUrl: './tile-header.component.html',
+  styleUrls: ['./tile-header.component.scss'],
+})
+export class TileHeaderComponent implements OnInit {
+
+  @Input()
+  public title: string | undefined;
+
+  @Input()
+  public data: DropdownModel[] = [];
+
+  @Output()
+  public moreButtonItemClicked: EventEmitter<DropdownModel> = new EventEmitter<DropdownModel>();
+
+  constructor() {
+  }
+
+  ngOnInit(): void {
+    this.data.push({
```

Comment:
I think it would be better if we have a separate ```dropdownOptions``` and we would not mutate input array. If the config would be saved in localstorage or database, mutating it would have bad effects.